OpenMage/magento-lts
# Added order tracking for Google Analytics 4

In https://github.com/OpenMage/magento-lts/issues/3090 it was pointed out that the previous UA/GA implementation supported the order data tracking, while the GA4 implementation isn't.

This PR aims to add the order tracking to GA4.

### Fixed Issues (if relevant)
1. Fixes https://github.com/OpenMage/magento-lts/issues/3090

### Manual testing scenarios (*)
1. Enable Analytics 4 in the backend configuration
2. Place an order
3. Check that there are no javascript errors and that the order data is sent to analytics
